src/main/java/se/sundsvall/comfactfacade/integration/comfact/configuration/ResponseResultCode.java

```diff
@@ -0,0 +1,82 @@
+	OPTIONAL_INPUT_NOT_SUPPORTED("OptionalInputNotSupported", 400),
+	SIGNING_INSTANCE_SYNCHRONIZATION_ERROR("SigningInstanceSynchronizationError", 500),
+	SIGNING_INSTANCE_NOT_FOUND("SigningInstanceNotFound", 404),
+	SERVICE_REQUEST_XML_VALIDATION_ERROR("ServiceRequestXMLValidationError", 400),
+	SIGNING_INSTANCE_XML_VALIDATION_ERROR("SigningInstanceXMLValidationError", 400),
+	SIGNING_INSTANCE_NOT_RUNNABLE("SigningInstanceNotRunnable", 500),
+	SIGNING_INSTANCE_COMPLETED("SigningInstanceCompleted", 200),
+	SIGNING_INSTANCE_WITHDRAWN("SigningInstanceWithdrawn", 200),
+	SIGNING_INSTANCE_APPROVED("SigningInstanceApproved", 200),
+	SIGNING_INSTANCE_DECLINED("SigningInstanceDeclined", 200),
+	SIGNING_INSTANCE_EXPIRED("SigningInstanceExpired", 200),
+	SIGNATORY_NOT_FOUND("SignatoryNotFound", 404),
+	SIGNATORY_APPROVED("SignatoryApproved", 200),
+	SIGNATORY_SYNCHRONIZATION_ERROR("SignatorySynchronizationError", 500),
+	INVALID_IDENTIFICATION_ALIAS("InvalidIdentificationAlias", 400),
+	INVALID_SERVICE_PROVIDER_CONFIGURATION("InvalidServiceProviderConfiguration", 500),
+	IDENTITY_PROVIDER_CONTRACT_NOT_FOUND("IdentityProviderContractNotFound", 404),
+	INVALID_IDENTITY_PROVIDER_CONTRACT("InvalidIdentityProviderContract", 400),
+	IDENTITY_PROVIDER_CONFIGURATION_NOT_FOUND("IdentityProviderConfigurationNotFound", 404),
+	INVALID_IDENTITY_PROVIDER_CONFIGURATION("InvalidIdentityProviderConfiguration", 400),
+	INVALID_IDENTIFICATION_TOKEN("InvalidIdentificationToken", 400),
+	INVALID_SIGNATURE_RESPONSE("InvalidSignatureResponse", 400),
+	IDENTIFICATION_TOKEN_NOT_AUTHENTICATED("IdentificationTokenNotAuthenticated", 500),
+	IDENTIFICATION_TYPE_DOES_NOT_SUPPORT_ACTION("IdentificationTypeDoesNotSupportAction", 500),
+	SIGNATURE_REQUEST_CREATION_FAILED("SignatureRequestCreationFailed", 500),
+	SIGNATURE_RESPONSE_SERVICE_ERROR("SignatureResponseServiceError", 500),
+	SIGNATURE_RESPONSE_USER_MISMATCH("SignatureResponseUserMismatch", 500),
+	SIGNATURE_RESPONSE_SIGN_MESSAGE_ERROR("SignatureResponseSignMessageError", 400),
+	SIGNATURE_TIMESTAMP_FAILED("SignatureTimestampFailed", 500),
+	SIGNATURE_CERTIFICATE_VERIFICATION_FAILED("SignatureCertificateVerificationFailed", 500),
+	SIGNATURE_TIMESTAMP_VERIFICATION_FAILED("SignatureTimestampVerificationFailed", 500),
+	AUGMENT_SIGNED_DOCUMENT_FAILED("AugmentSignedDocumentFailed", 500),
+	TRAFFIC_MESSAGE_FAILED("TrafficMessageFailed", 500),
+	CLIENT_SERVICE_UPDATE_FAILED("ClientServiceUpdateFailed", 500),
+	EMAIL_UPDATE_FAILED("EmailUpdateFailed", 500),
+	DOWN_FOR_MAINTENANCE("DownForMaintenance", 502);
+
+	private final String message;
+
+	private final int httpCode;
+
+	ResponseResultCode(final String message, final int httpCode) {
+		this.message = message;
+		this.httpCode = httpCode;
+	}
+
+	public static ResponseResultCode fromMessage(final String message) {
+		return Arrays.stream(values())
+			.filter(code -> code.message.equals(message)).findFirst()
+			.orElse(null);
```

Comment:
Add

`UNKNOWN_COMFACT_ERROR("UnknownComfactError", 500)`
and
`orElse(UNKNOWN_COMFACT_ERROR)`

to handle possible unmapped errors that might occur if Comfact decides to add new errors in the future?